### PR TITLE
Isomorphic

### PR DESCRIPTION
**Pull request by andresmoschini**
Hi @mravinale, @lucasasecas, @lmarquezMS and @matiasbeckerle,

I do not know if you are interested, if not, please ignore this message.

It allows to reuse modules on client and server side using CommonJS on server and generating a bundle with Browserify on client side.

If you are interested and have some time, please do a quick review.

### CODE REVIEW

**Comment by lmarquezMS** [comment]
Hi @andresmoschini.

I'm interested but I know a little bit about CommonJS so I will to read about that.
I'll try to review your code.

**Comment by andresmoschini** [comment]
No reviews, merging...

File: src/koclient/ko-view-model.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
-import Utils = require("./utils");
-import Model = require("./model");
+import * as utils from "../common/utils";
```
Comment (matiasbeckerle) [inline src/koclient/ko-view-model.ts:1]: Heeeeeeeeeeeeeeeeeeeeeere is what I was looking for! I could make it run and find this. Thank you.

Comment (andresmoschini) [inline src/koclient/ko-view-model.ts:1]: :blush: I am glad that you like it!

**Pull request merged**